### PR TITLE
Unit test

### PR DESCRIPTION
**Pull request by jessikka12**
css code cleanup and added a unit test for the annotation test

### CODE REVIEW

File: src/tests/annotation.test.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+import Annotation from "../components/Annotation.js";
+
+//test block
+test("annotation renders", () => {
+// render the component on virtual dom
+render(<Annotation num="1" title="Title" handleShowLocation={console.log('')} />);
+
+// var title = screen.getByRole("generic")
+
+//assert the expected result
```
Comment (charlesma11) [inline src/tests/annotation.test.js:11]: can we get some more assertions here?

**Review by charlesma11 — CHANGES_REQUESTED** [review]

File: src/tests/annotation.test.js (hunk 1 above)
Comment (jessikka12) [reply src/tests/annotation.test.js:9]: what kind of assertions were you thinking?

Comment (charlesma11) [reply src/tests/annotation.test.js:12]: make sure this works for edge cases (i.e. should go back to origin when a different number is inputted that is not in range)
also add for tests other than annotation

**Review by charlesma11 — CHANGES_REQUESTED** [review]

File: src/tests/annotation.test.js (hunk 1 above)
Comment (jessikka12) [reply src/tests/annotation.test.js:12]: ok, so we want a more expansive test suite. sure I can write more unit tests. 

but what do you mean by out of range and going back to origin? we don't have a range or an origin in the react code.

Comment (charlesma11) [reply src/tests/annotation.test.js:12]: if a user clicks on a the "view location" button, it takes them to an annotation with the "id" of the annotation number. Say the "id" sent is somehow a different number (like 100), then it should just return the origin position (which is 0). Basically testing the `handleShowLocation` method

Comment (jessikka12) [reply src/tests/annotation.test.js:12]: isn't the code for that on the unity side? the `handleShowLocation` method just directly passes the number into the `showMessage` method which is passed in from unity in App.js. We don't do any checks on the number because our annotations are out of order right now (they go 1, 11, 12, 20). 

```
  const handleShowLocation = (num) => {
    console.log(num);
    sendMessage('0_annotation_camera', 'SetLocation', num);
  };
```

also, checking that the "view location" button sends the player to the right location in the model is an integration test.

Comment (charlesma11) [reply src/tests/annotation.test.js:12]: Ok in that case no need to do the out of range test. We should add functionality on the front end to make sure the function doesn't pass in wrong numbers in the first place.

**Review by charlesma11 — APPROVED** [review]
